udata_hydra/migrations/main/20241016_add_parquet_columns.sql

```diff
@@ -0,0 +1,5 @@
+-- add parquet_url and parquet_size columns to resources/catalog table
+
+ALTER TABLE catalog
```

Comment:
I think `parquet` info should be stored in `checks` table as for `parsing_table` info

Reply:
Don't we want parquet files to be unique per resource, and replaced everytime there is a new analysis?

Reply:
I think having the information of the checks on which the parquet has been constructed is important.
If we want to move it, the logic should be the same as `parsing_table` in any case :)

Reply:
Done!

Reply:
For info this PR doesn't include the creation of a DB method helper to get the latest parquet url from a resource, I prefer having both PRs separated and it's not needed in the code yet. Let me know if it will be needed and I'll create a ticket about it

Reply:
:ok_hand: let's way for the need first indeed!